### DISCUSSION TITLE
Membership Payment Date Placerholder

### DISCUSSION THREAD

**Discussion by MichaelWybo** [liedekef/events-made-easy#22] [Q&A] [answered]
Hello Franky

In the back end we can consult the payment date of the membership. (or in my specific case, the date when i accept the payment manually)
Is there a placeholder which i can use to print it with the PDF Output ?

Thanks

**Comment by liedekef** [comment, marked as answer]
Indeed none was specifically present. Although there's a generic placeholder, this is better:
https://github.com/liedekef/events-made-easy/commit/4dcd7dcf1c776692ab14bdec9262ee519e43fbe6

**Comment by liedekef** [comment]
Ah, I just saw an error here. This is the fix (will be in the next version):
https://github.com/liedekef/events-made-easy/commit/d1f981a2bea74672d267eed45d395415c61b63a2#diff-54276e68c9cdceb37ff981bb5e5c30a39d0fb42192ee8ea679506983d5b3c51d

**Reply by MichaelWybo** [reply]
Haha I was just writing that i found this bug also :), I was hesitating to make a pull request; but saw that I didnt reinstalled IntelliJ on my computer.

**Reply by liedekef** [reply]
for small things like this, you can use github directly to edit a file. It will automatically create a fork in your home where you can edit any files. When finished you can then have github automatically create a pull request and even delete the fork when the merge is done.
